wp-content/themes/missilethreat/template-parts/entry-header.php

```diff
@@ -7,28 +7,137 @@
  * @since 1.0.0
  */
 
-$entry_header_classes = '';
 
+$page_for_posts = get_option( 'page_for_posts' );
 ?>
 
-<header class="single__header<?php echo esc_attr( $entry_header_classes ); ?>">
+<?php 
+if( is_single() && 'defsys' === get_post_type() || is_single() && 'missile' === get_post_type() ) { 
+	
+	$missile_short_name = get_field('missile_name');
 
-	<div class="single__header-wrapper">
+	?>
+	<header <?php post_class('single__header'); ?>>
 
+	<div class="single__header-wrapper">
 		<?php
-			missilethreat_posted_on();
+		get_template_part( 'template-parts/breadcrumbs' );
+
+		if ( $missile_short_name ) { ?>
+			<h1 class="single__header-title"><?php echo $missile_short_name; ?></h1>
+			<?php the_title( '<div class="single__header-excerpt">', '</div>' );
+		} 
+		
+		else {
+			the_title( '<h1 class="single__header-title">', '</h1>' );
+		} ?>
+
+		<div class="page__header-divider page__header-divider--thicc"></div>
+		
+		<div class="single__header-meta">
+			Last Updated <?php missilethreat_last_updated(); ?>
+		</div>
+
+		<?php  if( 'defsys' === get_post_type() ) { ?>
+			<div class="share-wrapper">
+				<?php missiledefense_system_terms(); ?>
+				<?php missiledefense_share(); ?>
+			</div>
+		<?php } ?>
 
-			the_title( '<h1 class="single__title">', '</h1>' );
+		</div><!-- .entry-header-inner -->
+	<?php } 
 
-			if ( has_excerpt() && is_singular() ) {
-				the_excerpt();
-			}
+	elseif ( 'systems' === get_post_type() || 'actors' === get_post_type() ) {
+		$feat_image = get_the_post_thumbnail_url( $post->ID ); ?>
+		<header class="page__header entry-header"  style="background-image: url('<?php echo $feat_image; ?>')">
+			<div class="overlay"></div>
```

Comment:
Unless there's a reason not too, I would use a pseudo element for the overlay. Or you can actually stack multiple background-images on top of each other. Take a look at the code generated by this [tool])(https://hero-generator.netlify.app/) as an example.

Reply:
wut. this thing is amazing!